src/jackknife.jl

```diff
@@ -0,0 +1,45 @@
+"""
+Use bootweights to creaye replicate weights using Rao-Wu bootstrap. The function accepts a `SurveyDesign` and returns a `ReplicateDesign` which has additional columns for replicate weights. 
+
+```jldoctest
+julia> using Random
+
+julia> dclus1 = SurveyDesign(apiclus1; clusters = :dnum, popsize=:fpc);
+```
+"""
+function jknweights(design::SurveyDesign; replicates = 10, rng = MersenneTwister(1234))

```

Comment:
JKn is supposed to be deterministic, you don't need a random number generator. Furthermore, the algorithm fixes the number of replicates, so that also shouldn't be a parameter.